src/game/data.py

```diff
@@ -88,6 +88,12 @@ def init_grid(self):
     def init_remaining_pieces(self):
         return [i + 1 for i in range(PIECES_NUMBER)]
 
+    def check_piece_validity(self, piece):
```

Comment:
What's piece validity ? If I don't understand the function name, it's maybe because the name is not adapted